GraphQL-TSC.md

```diff
@@ -209,9 +209,9 @@ Each repository will be managed by the EasyCLA signature tool, ensuring that all
 
 ### Speaking on behalf of the GraphQL Project
 
-GraphQL Contributors as individuals are welcome to express opinions and messages of political advocacy in any of their personal spaces (e.g. Twitter, blog) as well as within GitHub issues and PR discussions, provided it adheres to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md). However when speaking on behalf of the GraphQL project (e.g. technical docs, project websites, READMEs, or anything merged into a git repository under the GraphQL organization) we do not allow posting opinions or messages of political advocacy using the voice of the project without prior approval via GraphQL TSC vote.
+GraphQL Contributors as individuals are welcome to express opinions and messages of political advocacy in any of their personal spaces (e.g. personal Twitter, blog) as well as within GitHub issues and PR discussions, provided it adheres to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md). When speaking on behalf of the GraphQL project (e.g. technical docs, project websites, project Twitter, READMEs, or anything merged into a git repository under the GraphQL organization) we only allow posting opinions or messages of political advocacy using the voice of the project with prior approval via GraphQL TSC vote.
```

Comment:
The final sentence is a little repetative ("speaking on behalf [...] voice of the project") and the parenthesis in the middle makes it a little disjoint. I've had a go at editing it to address these concerns:

```suggestion
GraphQL Contributors as individuals are welcome to express opinions and messages of political advocacy in any of their personal spaces (e.g. personal Twitter, blog) as well as within GitHub issues and PR discussions, provided it adheres to our [Code of Conduct](https://github.com/graphql/foundation/blob/main/CODE-OF-CONDUCT.md). We require approval via GraphQL TSC vote prior to posting opinions or messages of political advocacy using the voice of the GraphQL project (e.g. within technical docs, project websites, project Twitter, READMEs, or anything merged into a git repository under the GraphQL organization).
```